targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_MCU_LPC546XX/CMakeLists.txt

```diff
@@ -94,4 +94,5 @@ target_link_libraries(mbed-lpc546xx
         mbed-lpc546xx-xpresso
 )
 
-mbed_post_build_lpc_patch_vtable()
+mbed_post_build_lpc_patch_vtable("LPC546XX")
+mbed_post_build_lpc_patch_vtable("FF_LPC546XX")
```

Comment:
looking at this from the cmake here, it would make sense to use the same name as CMake target.  In this case `mbed-lpc546xx-xpresso`, we do not need to have knowledge of targets.json naming in CMake.

We can fix this in separate PR, let's think about how and why. The naming could be CMake target name + post-build, this would result in `mbed-lpc546xx-xpresso-post-build`. This might be more obvious from CMake.